nixos/modules/services/misc/gotenberg.nix

```diff
@@ -0,0 +1,258 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.gotenberg;
+
+  args =
+    [
+      "--api-port=${toString cfg.port}"
```

Comment:
This should also use `--api-bind-ip=127.0.0.1` by default (configurable with an option).

The upstream default is to listen on all interfaces (`0.0.0.0`, see https://gotenberg.dev/docs/configuration), which is not a good safe default.

In general NixOS makes all services default to listen to localhost, and then the user can change it if desired: https://github.com/NixOS/nixpkgs/issues/100192#issuecomment-706633913

For a typical deployment of e.g. `paperless-ng`, it would also be very appropriate to listen to `localhost` only. The related `tika` service already does that right.

Reply:
good call, I'll make a PR to change that default.

Reply:
In my staged changes, waiting to resolve other issues.

Reply:
@pyrox0 Ah, `--api-bind-ip` was only merged last week and is part of [v8.12.0](https://github.com/gotenberg/gotenberg/releases/tag/v8.12.0), we don't have that version yet:

* https://github.com/gotenberg/gotenberg/pull/1010